src/PowerShellEditorServices.VSCode/CustomViews/CustomViewBase.cs

```diff
@@ -38,7 +38,7 @@ internal async Task CreateAsync()
                     Title = this.Title,
                     ViewType = this.ViewType,
                 }
-            );
+            ).ConfigureAwait(false);
```

Comment:
We should just return the result here rather than await it, and make the method non-`async`

Reply:
Why's that? And also, can we do it separately from fixing compiler warnings?

Reply:
> Why's that?

With an `async` method, the compiler transforms the method body into an enumerator, and `await` must be called, adding overhead in terms of task waiting and method execution. But in trivial cases, we're actually just creating an async task around awaiting another task. In those cases, the caller is going to have to await anyway, so may as well simply pass the task through. Compare this:

```csharp
        internal Task CreateAsync() =>
            languageServer.SendRequestAsync(
                NewCustomViewRequest.Method,
                new NewCustomViewRequest
                {
                    Id = this.Id,
                    Title = this.Title,
                    ViewType = this.ViewType,
                });
```

> can we do it separately from fixing compiler warnings?

Technically this is also a valid fix for this warning and just as simple

Reply:
Ah I see.